Classes/Controller/ConfigurationController.php

```diff
@@ -68,7 +68,7 @@ public function indexAction(): void
         $this->view->assignMultiple([
             'sitesConfiguration' => $this->settings['sites'],
             'managementAccounts' => $managementAccounts,
-            'flashMessages' => $this->flashMessageContainer->getMessagesAndFlush(),
+            'flashMessages' => $this->controllerContext->flashMessageContainer()->getMessagesAndFlush(),
```

Comment:
```suggestion
            'flashMessages' => $this->controllerContext->getFlashMessageContainer()->getMessagesAndFlush(),
```

Reply:
That`t my fault I guess... suggested that in the previous PR and did not think about that.
Sadly the PR seems to be not editable for us as core team.

@hphoeksma thanks again for the PR... you can enable on the PR creating that we are able to change the PR. Because without that we have no permission to commit to your branch.

Thanks in advance :)

Reply:
Thanks @kdambekalns for the catch ;-)